geospatial/include/gz/common/geospatial/Dem.hh

```diff
@@ -52,12 +60,12 @@ namespace gz
       /// \brief Indicate that this is a non Earth DEM.
       /// \param[in] _isNonEarthDem Should be true if this is a
       /// non earth DEM, otherwise false.
-      public: void SetNonEarthDEM(bool _isNonEarthDem);
+      public: void SetUnknownDEM(bool _isNonEarthDem);
 
       /// \brief Check if the loaded DEM is not from the Earth.
       /// \return True if the loaded DEM is from the Earth, otherwise
       /// returns False.
```

Comment:
```suggestion
      /// \brief Check if the loaded DEM has an unknown coordinate system.
      /// \return True if the loaded DEM has a known coordinate system, otherwise
      /// false.
```

Reply:
Done https://github.com/gazebosim/gz-common/pull/377/commits/e42c08d9e12ccf1902a20f930322f79a89d03238